avocado/core/teststatus.py

```diff
@@ -10,29 +10,18 @@
 # See LICENSE for more details.
 
 """
-Maps the different status strings in avocado to booleans.
-
-This is used by methods and functions to return a cut and dry answer to whether
-a test or a job in avocado PASSed or FAILed.
+Valid test statuses and whether they signal success (or failure).
 """
 
-mapping = {"SKIP": True,
-           "ABORT": False,
-           "ERROR": False,
-           "FAIL": False,
-           "WARN": True,
-           "PASS": True,
-           "START": True,
-           "ALERT": False,
-           "RUNNING": False,
-           "NOSTATUS": False,
-           "INTERRUPTED": False,
-           "CANCEL": True}
+#: Maps the different status strings in avocado to booleans.
+STATUSES_SUCCESS = {"SKIP": True,
```

Comment:
I am not quite sure if `STATUSES_SUCCESS` is the right name. It can indicate that is the statuses of tests which succeeded or passed. What about `STATUSES_MAPPING`?

Reply:
Naming is always hard.  I'm fine with `STATUSES_MAPPING` too.  I'll send a v2.